Sources/Turnstile/Credentials/Credentials.swift

```diff
@@ -13,4 +13,4 @@
  */
 public protocol Credentials { }
 
-public struct Sesssion: Credentials {}
+public struct Session: Credentials {}
```

Comment:
Just tested in playground and seems to work great!

<img width="634" alt="screen shot 2017-02-15 at 5 04 23 pm" src="https://cloud.githubusercontent.com/assets/7799267/23001125/d8d5acfa-f3a0-11e6-83e2-4f02db45b811.png">